env-var-docs/env_var_docs/schema/README.md

```diff
@@ -0,0 +1,97 @@
+# Environment variable documentation file format
+
+We maintain a [json-schema](https://json-schema.org/) that defines the expected
+structure of env-var-docs.json in [schema.json](./schema.json).
+
+The env-var-docs.json file contains key-value pairs. Each key is either a group
+of environment variables or an environment variable referenced in
+application.conf.
+
+Groups have the following fields:
+
+- `group-description`: A human-readable description of what the group
+  configures.
+- `members`: An object containing groups or variables.
+
+The `group-description` and `members` fields must be set for each group.
+
+Variables have the following fields:
+
+- `description`: A human-readable description of what the environment variable
+  configures.
+- `type`: The value type expected. Can be 'string', 'int', 'bool', or
+  'index-list'.
+- `required`: If the environment variable is required to be set. If the
+  `required` field is not set, the environment variable is not required.
```

Comment:
**question (blocking):** is it just checking if it's set or can it be `"required": false`?

Reply:
required can be set to false.  If the required key is not set, validation code should treat the variable as not required.